internal/main

```diff
@@ -510,8 +510,24 @@ function post_build_preview {
 
 # Publishes the docs for Jenkins or for a dev.
 function publish {
-    [ -n "$JENKINS_URL" ] && activate_jenkins_env
-    echo "where does publish live? in docker? or just here, like it is in fe lambda?"
+    if [ -n "$JENKINS_URL" ]; then
+	activate_jenkins_env
+	MESSAGE="Test, build, and publish with Jenkins (build # $BUILD_NUMBER)"
+    else
+	MESSAGE="Manually published by $USER"
+    fi
+    git worktree add dragons gh-pages
+    rm -rf dragons/*
+    rsync -a docs/_build/html/ dragons/
+    cd dragons
+    touch .nojekyll
```

Comment:
When I run this flow on my own, this file still exists in the dragons folder.

Reply:
Yeah, that's because the glob pattern `*` matches non-hidden files, meaning names that don't start with `.`. This is virtually unchanged from the way the Docs Starter Kit does it, so it should be good enough.